tests/tck/features/go/Orderby.feature

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Orderby Sentence
+
+  Background: Prepare space
+    Given a graph with space named "nba"
+
+  Scenario: Syntax Error
+    When executing query:
+      """
+      ORDER BY
+      """
+    Then a SyntaxError should be raised at runtime:
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name | ORDER BY $-.$$.team.name
+      """
+    Then a SyntaxError should be raised at runtime:
+    When executing query:
+      """
+      ORDER BY $-.xx
+      """
+    Then a SyntaxError should be raised at runtime:
+
+  Scenario: Empty Input
+    When executing query:
+      """
+      GO FROM "NON EXIST VERTEX ID" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team| ORDER BY $-.name
+      """
+    Then the result should be, in order, with relax comparison:
+      | name | start | team |
+    When executing query:
+      """
+      GO FROM "Marco Belinelli" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team
+      | YIELD $-.name as name WHERE $-.start > 20000 | ORDER BY $-.name
+      """
+    Then the result should be, in order, with relax comparison:
+      | name |
+
+  Scenario: Orderby not existing prop
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.abc
+      """
+    Then a SemanticError should be raised at runtime:
+
+  Scenario: Sorting results using single column
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.team
+      """
+    Then the result should be, in order, with relax comparison:
```

Comment:
In this feature file,  the `the result should be, in order` step is more reasonable.